### DISCUSSION TITLE
Release v2024.2

### DISCUSSION THREAD

**Discussion by jenriquesoriano** [INSPIRE-MIF/helpdesk-validator#1094] [Announcements]
Dear Community,

We are pleased to inform you that we are currently in the final stages of testing to ensure a seamless user experience.

We will keep you updated on our progress and provide a revised release date as soon as it becomes available, likely early next week.

In the meantime, if you have any questions or concerns, please do not hesitate to reach out to us.

Thank you.

Best regards,

**Comment by fabiovinci** [comment]
Dear Community,

just to inform you that the v2024.2 release has been deployed in the [Production instance](https://inspire.ec.europa.eu/validator/home/index.html).
For further details, please visit the [release notes](https://github.com/INSPIRE-MIF/helpdesk-validator/releases/tag/v2024.2).
